From afc5a46f121d8450ef5e8ca4f12ce9d0f72d283a Mon Sep 17 00:00:00 2001
From: Fei Jiang <fei.jiang@intel.com>
Date: Mon, 23 Jul 2018 10:51:33 +0800
Subject: [PATCH 1410/1586] drm/i915/gvt: enable plane update pvmmio through
 enable_pvmmio param

plane update pvmmio level is 0x2, need set it in enable_pvmmio for both SOS
and UOS kernel driver.
Patch for both SOS and UOS.

Signed-off-by: Fei Jiang <fei.jiang@intel.com>
Reviewed-by: Min He <min.he@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 040e32ae..1cfbdf1 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -24,6 +24,7 @@
 
 #include "i915_params.h"
 #include "i915_drv.h"
+#include "i915_pvinfo.h"
 
 #define i915_param_named(name, T, perm, desc) \
 	module_param_named(name, i915_modparams.name, T, perm); \
@@ -72,7 +73,7 @@ struct i915_params i915_modparams __read_mostly = {
 	.inject_load_failure = 0,
 	.enable_dpcd_backlight = false,
 	.enable_gvt = false,
-	.enable_pvmmio = 1,
+	.enable_pvmmio = PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE,
 	.enable_gvt_oos = 1,
 	.enable_conformance_check = true,
 	.disable_gvt_fw_loading = true,
-- 
2.7.4

